man/fi_av.3.md

```diff
@@ -328,6 +348,16 @@ that calls to fi_av_insert following a call to fi_av_remove always reference a
 valid buffer in the fi_addr parameter.  Otherwise it may be difficult to
 determine what the next assigned index will be.
 
+If the address vector is configured with authorization keys, the fi_addr
+parameter cannot be NULL and is used to define the authorization key associated
+with the address being inserted. Acceptable inputs are the fi_addr_t's generated
+from `fi_av_insert_auth_key`. Only `fi_addr[0]` can be used to pass in an
+authorization key fi_addr_t. Return returned fi_addr_t's will map to endpoint
+address against the specified authorization key. These fi_addr_t's can be
+used as the target for local data transfer operations. If the endpoint
+supports `FI_DIRECTED_RECV`, these fi_addr_t's can be used to restrict
```

Comment:
While I like the idea of reusing FI_DIRECTED_RECV to restrict recv buffers to a specific auth_key, I wonder if we need to restrict FI_DIRECTED_RECV to only be applicable if max_ep_auth_key_cnt <= 1 and define a new cap for this capability. Maybe FI_DIRECTED_AUTH_KEY_RECV?

Reply:
I think I am going to define FI_DIRECTED_AUTH_KEY_RECV for now as a discussion point.